modules/gplazma2-jython/src/main/resources/META-INF/gplazma-plugins.xml

```diff
@@ -0,0 +1,6 @@
+<plugins>
+  <plugin>
+    <name>Jython</name>
```

Comment:
Please make plugin name lowercase to match other plugins. Maybe `pyscript` (or similar) is a better name.  `Jython` tells too little to dCache admins.